public_html/wp-content/plugins/wordcamp-payments/includes/sponsor-invoice.php

```diff
@@ -514,27 +522,43 @@ function add_row_action( $actions, $post ) {
 		return $actions;
 	}
 
-	if ( 'wcbsi_uncollectible' === $post->post_status ) {
-		return $actions;
-	}
-
 	$post_type_object = get_post_type_object( $post->post_type );
 
-	$url = add_query_arg(
-		array(
-			'action' => 'wcbsi_update',
-			'status' => 'uncollectible',
-		),
-		admin_url( sprintf( $post_type_object->_edit_link, $post->ID ) )
-	);
+	if ( 'wcbsi_uncollectible' !== $post->post_status ) {
```

Comment:
Seems like `paid` and `refunded` invoices should also not have the option to mark as `uncollectible`...

Reply:
I didn't want to make changes to the behavior of uncollectible, and this is what it does currently. I can iterate on this in another PR, maybe?

Reply:
👍

Reply:
Update: sounds like `wcbsi_approved` is the only status when Uncollectible should be an option.